kiosk/src/Components/HighScoreInitial.tsx

```diff
@@ -31,10 +32,12 @@ const HighScoreInitial: React.FC<IProps> = ({ kiosk, isSelected, onCharacterChan
             if (!isSelected) { return; }
 
             if (kiosk.gamepadManager.isUpPressed()) {
+                tickEvent("kiosk.newHighScore.upPressed");
```

Comment:
This one seems bit too noisy. What are we learning from these ticks?

Reply:
I agree, it will be very noisy. I was wanting to use this to measure whether or not users were actually changing their initials when entering a high score. I have ticks where we know if the user is selecting letters (pressing A), which I think will provide enough context (and these can probably be edited to be less noisy as well). I was just curious if people would actually take the time to scroll through and find their initials. I'm not sure this data would change our approach to high scores in the future, though, so it might be a bit overkill.